Dockerfile

```diff
@@ -1,15 +1,5 @@
-FROM ubuntu:12.04
+FROM python:3.4-alpine
```

Comment:
Yes, I'd like to use popular distro too.

OK, then, why don't we use xenial (Ubuntu 16.04 LTS)? https://github.com/tnir/flower/blob/docker-xenial/Dockerfile

As the image from xenial has as big as 212MB, it's inconvenient for everyone to use it. Note that alpine-based flower image has just 49MB.

I'm providing the images on https://hub.docker.com/r/tnir/flower/tags/.